src/slack/slack.js

```diff
@@ -863,12 +864,8 @@ export class Slack {
       order.country = info.country
     }
 
-    // TODO: set fieldTitle according to order step/state
-    let fieldsTitle
-    if (!order.country) {
-      fieldsTitle = 'In which country is your office?'
-    } else if (order.isHome === null) {
-      fieldsTitle = 'Where do you want to pickup the order?'
+    if (order.step === 'new') {
+      order.step = !order.country ? 'country' : 'delivery'
```

Comment:
so, regarding the Wincent flow as mentioned in another comment... we should change step here to `type`. I thought we could create something like `INITIAL_STEP` constant for all the variants and get it here, but then again there's this handling of `country`/`delivery` if country was not detected from link... so maybe just throw a line for wincent here